packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -457,12 +505,16 @@ - (BOOL)setCaptureSessionPreset:(FLTResolutionPreset)resolutionPreset withError:
         _videoCaptureSession.sessionPreset = AVCaptureSessionPresetLow;
         _previewSize = CGSizeMake(352, 288);
       } else {
-        *error = [NSError errorWithDomain:NSCocoaErrorDomain
-                                     code:NSURLErrorUnknown
-                                 userInfo:@{
-                                   NSLocalizedDescriptionKey :
-                                       @"No capture session available for current capture session."
-                                 }];
+        // Assign passed by reference `error` only if requested by caller.
```

Comment:
This is a standard Obj-C pattern, so does not need a comment explaining it.

Reply:
comment removed